src/vs/workbench/contrib/chat/electron-sandbox/actions/voiceChatActions.ts

```diff
@@ -54,18 +54,24 @@ import { IVoiceChatService } from 'vs/workbench/contrib/chat/common/voiceChat';
 import { IKeybindingService } from 'vs/platform/keybinding/common/keybinding';
 import { ThemeIcon } from 'vs/base/common/themables';
 import { EditorContextKeys } from 'vs/editor/common/editorContextKeys';
+import { ITerminalService } from 'vs/workbench/contrib/terminal/browser/terminal';
+
+// This is a one-off/safe import, changing the eslint rules would require duplicating/complicating the rules
+// eslint-disable-next-line local/code-import-patterns
+import { TerminalChatController } from 'vs/workbench/contrib/terminalContrib/chat/browser/terminalChatController';
```

Comment:
Would it help if `vs/workbench/contrib/terminal` or similar somehow re-exported the `TerminalChatController` so that we can avoid this ESLint issue?

Reply:
@bpasero the terminal has its own contrib model similar to the editor that is enforced by eslint where `contrib/terminalContrib` cannot be imported from generally. In order to do what you say we'd need to throw away all the benefits we get from this and move the chat portion into `contrib/terminal` where it's no longer a self-contained extension to the terminal, enforced at runtime.

But yeah, we could re-export from terminal so this exception comment can be contained inside `contrib/terminal` instead of leaking outside.